openedx/features/course_experience/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Unified course experience settings
+
+UNIFIED_COURSE_EXPERIENCE_FLAG = 'unified_course_experience'
+UNIFIED_COURSE_VIEW_FLAG = 'unified_course_view'
```

Comment:
I hate calling things new, because before you know it that thing is no longer new. :-)